azure-pipelines-v3.yml

```diff
@@ -40,22 +40,31 @@ jobs:
       arguments: -warnAsError -c Release
 
   # dotnet test
-  - task: DotNetCoreCLI@2
+  - task: AzureCLI@2
     displayName: dotnet test
     condition: and(succeeded(), ne(variables['warm-up'], 'true'))
     inputs:
-      command: test
-      arguments: -c Release --no-build /p:CollectCoverage=true /p:CoverletOutputFormat=cobertura
-      publishTestResults: true
+      azureSubscription: 'DevRel - Docs Build Worker (PubDev)'
+      scriptType: ps
+      scriptLocation: inlineScript
+      inlineScript: |
+        dotnet test -c Release --no-build /p:CollectCoverage=true /p:CoverletOutputFormat=cobertura --logger trx --results-directory ./_temp/testResult
     env:
       DOCS_GITHUB_TOKEN: $(gitHubTestToken)
       DOCS_OPS_TOKEN: $(opBuildUserToken)
       DOCFX_MICROSOFT_GRAPH_TENANT_ID: '72f988bf-86f1-41af-91ab-2d7cd011db47'
       DOCFX_MICROSOFT_GRAPH_CLIENT_ID: 'b799e059-9dd8-4839-a39c-96f7531e55e2'
       MICROSOFT_GRAPH_CLIENT_CERTIFICATE: $(Docfx-MS-Graph-Client-Cert-PubDev)
+  
+  - task: PublishTestResults@2
+    inputs:
+      displayName: Publish Test Results
+      testResultsFormat: VSTest
+      testResultsFiles: '**/cloudtest_*.trx'
```

Comment:
Why `cloudtest_`?
```suggestion
      testResultsFiles: '**/*.trx'
```

Reply:
![image](https://user-images.githubusercontent.com/42199316/129167285-425d62e3-499e-456b-8aa8-5ae6706f46a7.png)
Seems all `dotnet test` result in CI run is prefixed with this, lemme try if this can be specified.

Reply:
with `dotnet test -c Release --no-build --logger "trx;LogFileName=TestResults.trx" --results-directory ./_temp`, one result file will overwrite another.
well, may have to `foreach *.csproj` to specify the result file name for each single test project.

Simplified as suggested.